capabilities.json

```diff
@@ -92,447 +88,202 @@
-                        "value": "Cambria"
-                    },
-                    {
-                        "displayName": "Cambria Math",
-                        "value": "Cambria Math"
-                    },
-                    {
-                        "displayName": "Candara",
-                        "value": "Candara"
-                    },
-                    {
-                        "displayName": "Comic Sans MS",
-                        "value": "Comic Sans MS"
-                    },
-                    {
-                        "displayName": "Consolas",
-                        "value": "Consolas"
-                    },
-                    {
-                        "displayName": "Constantia",
-                        "value": "Constantia"
-                    },
-                    {
-                        "displayName": "Corbel",
-                        "value": "Corbel"
-                    },
-                    {
-                        "displayName": "Courier New",
-                        "value": "Courier New"
-                    },
-                    {
-                        "displayName": "Georgia",
-                        "value": "Georgia"
-                    },
-                    {
-                        "displayName": "Lucida Sans Unicode",
-                        "value": "Lucida Sans Unicode"
-                    },
-                    {
-                        "displayName": "Symbol",
-                        "value": "Symbol"
-                    },
-                    {
-                        "displayName": "Tahoma",
-                        "value": "Tahoma"
-                    },
-                    {
-                        "displayName": "Times New Roman",
-                        "value": "Times New Roman"
-                    },
-                    {
-                        "displayName": "Trebuchet MS",
-                        "value": "Trebuchet MS"
-                    },
-                    {
-                        "displayName": "Verdana",
-                        "value": "Verdana"
-                    },
-                    {
-                        "displayName": "Wingdings",
-                        "value": "Wingdings"
-                    }
-                ]
             }
           }
+        },
+        "fontSize": {
+          "type": {
+            "formatting": {
+              "fontSize": true
+            }
+          }
+        },
+        "fontFamily": {
+          "type": {
+            "formatting": {
+              "fontFamily": true
+            }
+          }
+        },
+        "bold": {
+          "type": {
+            "bool": true
+          }
+        },
+        "italic": {
+          "type": {
+            "bool": true
+          }
+        },
+        "underline": {
+          "type": {
+            "bool": true
+          }
+        },
+        "forceDisplay": {
+          "description": "Display all labels anyway",
```

Comment:
DescriptionKey also can be moved into formatting settings.

Reply:
I have removed it [e0699b9](https://github.com/microsoft/powerbi-visuals-heatmap/pull/118/commits/e0699b961a6b8a0713a70463e05382c0843a4e1f). That specific capabilities object has no corresponding formatting settings input tho, probably because it does the same thing as the existing "show" input.